### PR TITLE
add new cat breeds and fix dog breed classification

### LINKED ISSUE #161: Additional cat breeds
Dear Katie,

can we please add three cat (felis catus) breeds to VBO:

Altai (synonym Altay)
Topaz 
Celestial

All are mentioned in a recent paper: PMID: [38644700](https://pubmed.ncbi.nlm.nih.gov/38644700/) and I found some mention in the following website which may not be suitable as a soruce http://messybeast.com/blue-eye-breeds.htm

Regards,

Imke

### PR DESCRIPTION
**Pull request by katiermullen**
addresses #160 - leaving open because @ImkeTammen may have additional comments about classification of German Pointer dogs.

closes #161

### CODE REVIEW

File: src/ontology/components/catbreeds.owl [hunk 1]
```diff
@@ -14812,6 +14812,127 @@
...
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0100260 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0100260">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NCBITaxon_9685"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0400000"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100259"/>
```
Comment (sabrinatoro) [inline src/ontology/components/catbreeds.owl:14862]: "has foundation stock" SubClass_Of is missing a source

File: src/ontology/components/catbreeds.owl [hunk 2]
```diff
@@ -14812,6 +14812,127 @@
...
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100051"/>
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100052"/>
```
Comment (sabrinatoro) [inline src/ontology/components/catbreeds.owl:14906]: 'has foundation stock" missing a source

File: src/ontology/components/catbreeds.owl [hunk 3]
```diff
@@ -14812,6 +14812,127 @@
...
+
+    <!-- http://purl.obolibrary.org/obo/VBO_0100261 -->
+
+    <owl:Class rdf:about="http://purl.obolibrary.org/obo/VBO_0100261">
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/NCBITaxon_9685"/>
+        <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/VBO_0400000"/>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100051"/>
```
Comment (sabrinatoro) [inline src/ontology/components/catbreeds.owl:14900]: 'has foundation stock" missing a source

File: src/ontology/components/catbreeds.owl [hunk 4]
```diff
@@ -14812,6 +14812,127 @@
...
+            </owl:Restriction>
+        </rdfs:subClassOf>
+        <rdfs:subClassOf>
+            <owl:Restriction>
+                <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/VBO_0300019"/>
+                <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/VBO_0100052"/>
+            </owl:Restriction>
```
Comment (sabrinatoro) [inline src/ontology/components/catbreeds.owl:14907]: Also double checking that this breed "Celestial (cat)" has 2 foundation stock: british shorthair and british longhair

**Review by sabrinatoro — CHANGES_REQUESTED** [review]
@katiermullen 
It looks good except for the fact that we should have a source for the "has foundation stock" annotations. 
Note that we do not have sources for "felis catus" SubClass of and for "canis lupus familiaris" (which is not ideal, but we could easily add the source for dog breed). However, I think we absolutely need to make sure that we have a source for the "has foundation stock" annotations to make sure we know where the information comes from.

File: src/ontology/components/catbreeds.owl (hunk 2 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14906]: The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

File: src/ontology/components/catbreeds.owl (hunk 3 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14900]: The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

File: src/ontology/components/catbreeds.owl (hunk 4 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14907]: Yes, that is correct.

File: src/ontology/components/catbreeds.owl (hunk 2 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14906]: The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

File: src/ontology/components/catbreeds.owl (hunk 1 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14862]: The source is a PMID. I put it in a column labelled ">A oboInOwl:source SPLIT=|" I am not sure why it is not showing?

Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14862]: done

File: src/ontology/components/catbreeds.owl (hunk 2 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14906]: done

File: src/ontology/components/catbreeds.owl (hunk 3 above)
Comment (katiermullen) [reply src/ontology/components/catbreeds.owl:14900]: done

**Review by sabrinatoro — APPROVED** [review]
It looks good! Approving and merging. Thank you @katiermullen.

**Pull request merged**